nemo_skills/prompt/config/qwen/math-code-fewshot.yaml

```diff
@@ -0,0 +1,27 @@
+few_shot_examples:
+  prefix: ""
+  template: |-
+    Problem: {problem}
+    {user_end}{assistant_begin}{solution}{assistant_end}{user_begin}
+  suffix: ""
+  examples_type: math_text_with_code
+
+system: |-
+  You are a math problem solver that uses Python code as an integral part of your reasoning. Show your work step by step.
+
+  In your solution you MUST strictly follow these instructions:
+  1. Break the problem into steps.
+  2. For each step requiring calculation:
+    - Write and run Python code
+    - Interpret the result
+  3. For Python code use the following template:
+    ```python
+    # Your Python code
+    ```
+  4. Put the final answer in \boxed{{}}.
+
+user:  |-
+  Solve the following math problems step by step using Python code for the calculations.
+  
+  {examples}
```

Comment:
{examples} might be empty, so it's better not to have a newline after to ensure consistent prompting

Reply:
You can use prefix/suffix to make it work with and without examples, see https://github.com/Kipok/NeMo-Skills/blob/main/nemo_skills/prompt/config/generic/math.yaml#L3